pkg/scheduler/scheduler.go

```diff
@@ -110,8 +110,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		c := snapshot.ClusterQueues[e.ClusterQueue]
 		if len(e.borrows) > 0 && c.Cohort != nil && usedCohorts.Has(c.Cohort.Name) {
 			e.admissible = false
```

Comment:
it should be `e.admitted`

Reply:
during nominate, it would only be admissible not admitted. I would keep this variable name.